### PR TITLE
ci: add yml validation test

### PR DESCRIPTION
**Pull request by cubercsl**
Add a yml validation test in pull request.

### CODE REVIEW

File: conference-yaml-schema.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+              - UTC-10
+              - UTC-9
+              - UTC-8
+              - UTC-7
+              - UTC-6
+              - UTC-5
+              - UTC-4
+              - UTC-3
+              - UTC-2
+              - UTC-1
```
Comment (0x4f5da2) [inline conference-yaml-schema.yml:102]:
```suggestion
              - UTC-1
              - UTC-0
```

Comment (0x4f5da2) [inline conference-yaml-schema.yml:102]:
```suggestion
              - UTC-1
              - UTC-0
```

**Review by 0x4f5da2 — APPROVED** [review]
LGTM

File: conference-yaml-schema.yml (hunk 1 above)
Comment (cubercsl) [reply conference-yaml-schema.yml:102]: According to https://github.com/ccfddl/ccf-deadlines/pull/77, there are only `UTC` and `UTC+0` in our data now.

**Pull request merged**